### PR TITLE
use log2 odds ratio in Mutex

### PR DESCRIPTION
**Pull request by dippindots**
change from log to log2, add the tooltip.

### CODE REVIEW

**Comment by alisman** [comment]
this PR is dependent on 1791, correct?

**Comment by dippindots** [comment]
@alisman Yes, this is dependent on 1791.

File: src/pages/resultsView/ResultsViewPageStore.ts [hunk 1]
```diff
@@ -302,6 +305,44 @@ export function extendSamplesWithCancerType(samples:Sample[], clinicalDataForSam
 
 }
 
+export function getSampleAlteredMap(filteredAlterationData: IQueriedMergedTrackCaseData[], samples: Sample[], oqlQuery: string){
```
Comment (alisman) [inline src/pages/resultsView/ResultsViewPageStore.ts]: sorry this should go in ResultsViewPageStoreUtils

File: src/pages/resultsView/ResultsViewPageStore.ts [hunk 2]
```diff
@@ -302,6 +305,44 @@ export function extendSamplesWithCancerType(samples:Sample[], clinicalDataForSam
...
+        return notGroupedOql.gene;
+    }
+    //gene with alteration type
+    else {
+        return notGroupedOql.oql_line.slice(0, -1);
+    }
+}
+
+export function getMultipleGeneResultKey(groupedOql: MergedTrackLineFilterOutput<AnnotatedExtendedAlteration>){
+    return groupedOql.label! === undefined ? _.map(groupedOql.list, (data) => data.gene).join(' / ') : groupedOql.label!;
```
Comment (alisman) [inline src/pages/resultsView/ResultsViewPageStore.ts]: you shouldn't need the ! if you are checking whether it's defined b/c typscript should not compain

File: src/pages/resultsView/ResultsViewPageStore.ts (hunk 1 above)
Comment (alisman) [reply src/pages/resultsView/ResultsViewPageStore.ts]: and this one should have a unit test.

**Pull request merged**